src/coreclr/scripts/superpmi_diffs_setup.py

```diff
@@ -211,6 +234,7 @@ def main(main_args):
     Note:
     1. asmdiffs uses Checked JITs, tpdiff uses Release JITs. Only the one needed is copied to the payload directory.
     2. Only asmdiffs needs jit-analyze and git
+    3. tpdiff can run on Linux, but asmdiffs is not implemented to run on Linux
```

Comment:
What is the problem with asmdiffs?

Reply:
Is it because of the availability of cordistools?

Reply:
Not cordistools. It's because we need jit-analyze and `git`. In particular, we need `git` on the Helix machine so it can be used for `git diff` by jit-analyze. For Windows, we download a "portable git" package to the correlation payload. Building jit-analyze for Linux wouldn't be hard, but would require a few tweaks.

However, bottom-line: we don't need to run diffs natively on Linux. The results would be (should be!) identical to running them on Windows.

Reply:
One (obvious) benefit would be to allow contributors who work on Linux systems to run asmdiffs locally. I think we should consider investing in it since it doesn't sound like it would be too expensive.

Reply:
I opened #86181 for it

Reply:
> allow contributors who work on Linux systems to run asmdiffs locally.

can't you already do that? I think @BruceForstall is referring to the problems we encounter when doing it in CI.

Reply:
Not sure? The comment seemed a bit general

> asmdiffs is not implemented to run on linux

If it's just in CI then :+1:

Reply:
Yeah, this was in the context of CI (and @BruceForstall  can correct me). But to run `asmdiff`, you need:
1. baseline jit (we already have that for native binaries like `libclrjit.so`)
2. cordistools
3. jit-analyze, hopefully this would just work cross plat.

Reply:
Ah, this comment is not in superpmi.py, but rather in the pipeline python script, so `asmdiffs` here probably refers to the pipeline job and not superpmi command?

Reply:
Correct.